backend/controllers/bitbucket.js

```diff
@@ -0,0 +1,51 @@
+const axios = require('axios');
+const urlList = require('../config/urlList');
+
+const bitbucket = async username => {
+	let result = {};
+	if (username) {
+		await axios
+			.get(urlList.bitbucket + username, {
```

Comment:
```suggestion
			.head(urlList.bitbucket + username, {
```

Reply:
Get Request
![image](https://user-images.githubusercontent.com/54680709/195249547-6b758b32-6a98-4ca1-8dd4-e4a30e08f9ae.png)
Head Request
![image](https://user-images.githubusercontent.com/54680709/195249627-eb296419-b1d8-4b40-a052-27cbe93d7582.png)
Hi @hoonn9 could you please replace get with head request as it will reduce payload by huge factor.

Reply:
That's right. I fixed it from get method to head method. Thank you for the review.